docs/requirements.txt

```diff
@@ -0,0 +1 @@
+breathe
```

Comment:
what is this?

Reply:
breathe is a sphinx plugin for doxygen...which may not help clarify things too much"

It exposes some additional Sphinx directives such as:
https://raw.githubusercontent.com/brightcove/valerie/master/docs/api.rst

It can also generate all such documentation automatically but that's avoided for additional control to start (particularly since doxygen isn't inherently Java oriented).

This is used to generate the API docs from the Java Docs via doxygen:
http://valerie.readthedocs.io/en/docs/api.html

Reply:
https://breathe.readthedocs.io/en/latest/